src/Rings/mpolyquo-localizations.jl

```diff
@@ -1525,3 +1525,17 @@ end
 @attr function is_prime(I::MPolyQuoLocalizedIdeal)
   return is_prime(saturated_ideal(I))
 end
+
+@attr function is_integral_domain(W::MPolyQuoLocalizedRing)
+  return is_prime(modulus(W))
+end
+
+@Markdown.doc """
+    is_integral_domain(R::Ring)
+
+Return whether or not `R` is an integral domain.
+"""
+function is_integral_domain(R::Ring)
+  is_domain_type(typeof(R)) && return true
+  error("method not implemented for rings of type $(typeof(R))")
+end
```

Comment:
@thofma @fieker @fingolfin : This would be my implementation of the `is_integral_domain` thing. The other (implemented) methods are scattered throughout the files in this commit.